installer.iss

```diff
@@ -82,10 +85,49 @@ spanish.ConfirmUninstall=¿Seguro que desea desinstalar {#RetroBarName}?
 spanish.WizardUninstalling=Estado de la desinstalación
 spanish.TranslatorNote=Updated Spanish translation courtesy of Amaro Martínez for {#RetroBarPublisher}.
 
+polish.StopDownload=Czy zatrzymać pobieranie?
+polish.ExitSetupTitle=Wyjdź z instalacji
+polish.SelectLanguageTitle=Wybierz język instalacji
+polish.WizardSelectTasks=Wybierz zadania dodatkowe
+polish.WizardSelectDir=Wybierz folder docelowy
+polish.BrowseDialogTitle=Instalowanie: wybierz folder
+polish.WizardReady=Gotowy do instalacji
+polish.WizardPreparing=Przygotowanie instalacji
+polish.ConfirmUninstall=Czy na pewno chcesz odinstalować {#RetroBarName}?
+polish.WizardUninstalling=Stan odinstalowywania
+polish.TranslatorNote=Zaktualizowane tłumaczenie polskie dla: {#RetroBarPublisher}.
+
+russian.StopDownload=Остановить загрузку?
+russian.ExitSetupTitle=Выйти из установки
+russian.SelectLanguageTitle=Выбрать язык установки
+russian.WizardSelectTasks=Выбрать дополнительные задачи
+russian.WizardSelectDir=Выбрать целевую папку
+russian.BrowseDialogTitle=Установка: выберите папку
+russian.WizardReady=Готов к установке
+russian.WizardPreparing=Подготовка установки
+russian.ConfirmUninstall=Вы уверены, что хотите удалить {#RetroBarName}?
+russian.WizardUninstalling=Состояние удаления (деинсталляции)
+russian.TranslatorNote=Обновлённый русский перевод для: {#RetroBarPublisher}.
+
+french.StopDownload=Arrêtez-vous le téléchargement?
+french.ExitSetupTitle=Quitter l'installation
+french.SelectLanguageTitle=Choisir la langue d'installation
+french.WizardSelectTasks=Choisir des tâches supplémentaires
+french.WizardSelectDir=Choisir le dossier pour l'installation
+french.BrowseDialogTitle=Installation: choisissez le dossier
+french.WizardReady=Prêt pour l'installation
+french.WizardPreparing=Préparation de l'installation
+french.ConfirmUninstall=Êtes-vous sûr de vouloir désinstaller? {#RetroBarName}?
+french.WizardUninstalling=Statut de désinstallation
+french.TranslatorNote=Traduction française mise à jour pour: {#RetroBarPublisher}.
```

Comment:
Please do not retranslate strings if they do not have typos, you can check if the language you are translating has correct grammar here: https://github.com/jrsoftware/issrc/tree/main/Files/Languages
If not, modifying the strings you need can help make the translation feel consistent and more appropriate.